.gitignore

```diff
@@ -46,6 +46,9 @@ npm-debug.log
 npm*
 *.gz
 
+# we'll ignore lock files as we are a public repo
+yarn.lock
```

Comment:
Might as well add npm's lock file too.

Reply:
Both are anyway included in .gitignore (line 62 and 63)